docs/roles/president.rst

```diff
@@ -9,22 +9,48 @@ This page explains more about the responsibilities and role of the president.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
+The president acts as a leader for the club to help determine strategy and focus for each academic year.
+The president uses the `RITlug/tasks <https://github.com/RITlug/tasks>`_ repository as their primary organizational tool. 
+This includes updating the `RITlug project board <https://github.com/orgs/RITlug/projects/1?fullscreen=true>`_
+The president works together with the eboard to work towards the goals and strategies for each semester. 
+Additionally, they can pick up tasks from other eboard members if they are overloaded.
+
+While it is the job of the president to carry out the main administrative tasks listed above, it is the vice president's job to assist where possible. 
+These items are the primary responsibilities of the president.
+The president is the main point of contact for all club-related functions.
 
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-< basic eligiblity requirements – what if someone is on co-op? away from campus? working remote? is under academic suspension (i.e. low GPA)? >
-
+* Must be involved in the club for at least a year prior to election
+* Must be a full time student on main campus for at least one semester
+* Cannot be on academic probation
+* Must be available for **75% of weekly club meetings and eboard meetings**
```

Comment:
Make sure by "available" you really mean "available" rather than "present". Might also be worth a quick sentence to say what "available" means.